src/types/componentAction.ts

```diff
@@ -1,6 +1,7 @@
+import { ReactElement } from 'react';
 import { ButtonProps } from '../@next';
 
 export type ComponentAction = ButtonProps & {
-  label: string;
+  label: ReactElement;
```

Comment:
can you use React.Node instead? since we want to allow any valid react child

Reply:
fixed: https://github.com/glints-dev/glints-aries/pull/901/commits/bbc61f95170923680be4259e18f3b2bd2428e4f9